openssf-compiler-options/usr/local/bin/gcc-wrapper

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/$(basename $0) -specs openssf.spec "$@"
```

Comment:
avoid a fork that will be hit thousands (millions?) of times a day  with `/usr/bin/${0##*/}`